guides/whitelist.htm

```diff
@@ -247,3 +250,8 @@ <h3 id="about-gdpr" class="anchor-adjust">About GDPR</h3>
     
 
 </section>
+<div class="container">
+    <p>GeForce is a trademark, or registered trademark, of NVIDIA Corporation.</p>
+    <p>Radeon is a trademark, or registered trademark, of Advanced Micro Devices, Inc.</p>
+    <p>Arc is a trademark, or registered trademark, of Intel Corporation.</p>
+</div>
```

Comment:
Maybe worth putting a small header here and just making this one `<p>` tag  since it looks a little strange so space out (at least to me)

![image](https://github.com/gridcoin-community/Gridcoin-Site/assets/30132912/6eca983f-3496-4d1b-8725-a21dfd5bc4fd)